training/ilab-wrapper/ilab

```diff
@@ -18,6 +18,32 @@ function verify_range() {
     fi
 }
 
+check_insights() {
+    if [[ -f /etc/insights-client/machine-id ]]; then
+        return
+    fi
+    if [[ -f /etc/ilab/skip-insights-validation ]]; then
```

Comment:
make this file name "insights-opt-out" so its more explicit what the user is doing.

Reply:
Sounds good to me